Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material

```diff
@@ -2,23 +2,23 @@
     "materialType": "@gemroot:Atom_Feature_Common@/Assets/Materials/Types/StandardPBR.materialtype",
```

Comment:
I can't find any place where this material is used. RosRobot uses this one: `RosRobotSample/Assets/robot/Rosbot_Mat.material` (which, BTW, looks identical). If it's not used, we can remove it.

Reply:
@zakmat could you remove this one as well?

Reply:
All right, I have removed it